.jvmopts-ci-17

```diff
@@ -0,0 +1,6 @@
+# This is used to configure the sbt instance that GH workflows launch
+
+-Xms1G
+-Xmx1G
+-Xss2M
+--add-opens=java.base/java.util.concurrent=ALL-UNNAMED
```

Comment:
Would be better to add this to the build so that tests will pass also when running locally with JDK 17. See Akka build: https://github.com/akka/akka/blob/main/project/JdkOptions.scala#L32

Reply:
My bad. Not needed.